zenodo/modules/spam/templates/zenodo_spam/safelist/admin.html

```diff
@@ -52,6 +52,14 @@ <h1>Safelist admin</h1>
                  ('max_users') }}">
         </div>
       </div>
+      <select name="include_pending" class="form-control">
+        <option id="1" {{ "selected" if  request.args.get
```

Comment:
```suggestion
        <option value="1" {{ "selected" if  request.args.get
```

maybe works with GET

Reply:
It does work with GET btw :)